packages/pass-style/src/types.d.ts

```diff
@@ -85,14 +85,14 @@ export type Passable<
 > = void | Primitive | Container<PC, E> | PC | E;
 
 export type Container<PC extends PassableCap, E extends Error> =
-  | CopyArrayI<PC, E>
-  | CopyRecordI<PC, E>
-  | CopyTaggedI<PC, E>;
-interface CopyArrayI<PC extends PassableCap, E extends Error>
+  | CopyArrayCommon<PC, E>
```

Comment:
this really is an interface version of the CopyArray type.

Would you be okay with `CopyArrayInterface`?

Reply:
Yes. Done, thanks.